### PR TITLE
Add clean.sh

### PR DESCRIPTION
**Pull request by steadyk**
To avoid removing the whole Kind cluster with all images, we add a `clean.sh` script which can be used to clean up the cluster resources. Together with `init.sh` this can be used to restore the riddles.

### CODE REVIEW

**Comment by MarDipp** [comment]
The new script description is missing in the screenshot on README.md

**Comment by MarDipp** [comment]
Wouldn´t a name like "reset.sh" be better?
The goal of the script is to reset the cluster to factory settings. The name "clean" doesn´t say that directly.

File: welcomemessage.sh [hunk 1]
```diff
@@ -19,6 +20,7 @@ printf "${CYAN}
 printf "${CYAN}                                                                         |_|                                      ${NC}\n"
 printf "\n"
 printf "To initialize a new local cluster run '${GREEN}. init.sh${NC}'.\n"
+printf "To cleanup resources run '${YELLOW}./clean.sh${NC}'.\n"
```
Comment (bavaria-blue) [inline welcomemessage.sh:23]: Suggestion:
printf "To cleanup resources run '${YELLOW}./clean.sh${NC}'.\n"
=>
printf "To remove most resources (without deleting the cluster) run '${YELLOW}./clean.sh${NC}'.\n"

**Comment by steadyk** [comment]
> The new script description is missing in the screenshot on README.md

Good hint! I will do this if all questions are resolved! :-)

File: welcomemessage.sh (hunk 1 above)
Comment (steadyk) [reply welcomemessage.sh:23]: Done!

**Comment by bavaria-blue** [comment]
> > Wouldn´t a name like "reset.sh" be better? The goal of the script is to reset the cluster to factory settings. The name "clean" doesn´t say that directly.
> 
> Hm, the script itself doesn't reset the cluster but remove all occurrences of some selected resource types from the cluster. To reset the cluster `./clean.sh` and `. init.sh` needs to be executed.
> 
> Is `clean.sh` then a proper name? Or should we consider creating a `reset.sh` script which resets the cluster?

Hmm, maybe:
clean.sh => remove-resources.sh
delete.sh => delete-cluster.sh
?

**Comment by steadyk** [comment]
@MarDipp @bavaria-blue
I've tried to adapt the naming as suggested. Maybe you could check again if this is fine now?

**Review by MarDipp — APPROVED** [review]
👍  Now the names sound better to me.

Resetting the riddles with the script works like a charm.

**Review by bavaria-blue — APPROVED** [review]
lgtm :+1:

**Pull request merged**